src/main/java/bsh/Interpreter.java

```diff
@@ -825,6 +827,15 @@ public final void print( Object o )
         }
     }
 
+    @Override
+    public final void prompt(String prompt) {
+        if (console != null) {
+            console.prompt(prompt);
+        } else {
+            print(prompt);
+        }
+    }
+
     // End ConsoleInterface
```

Comment:
Lets not implement this here, would you agree?

We can use the default method for now and avoid creating another public api method on Interpreter which we can't delete. Once we implement the default console class we can elaborate on that. For now the inherited method from `ConsoleInterface` which delegates to print will suffice.

So please remove this method implementation.